py3-cassandra-driver.yaml

```diff
@@ -0,0 +1,39 @@
+# Generated from https://pypi.org/project/cassandra-driver/
+package:
+  name: py3-cassandra-driver
+  version: 3.28.0
+  epoch: 0
+  description: DataStax Driver for Apache Cassandra
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - py3-six
+      - py3-geomet
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: e3400a1e1eeb34fc1162fd601bf667cf802761ab
+      repository: https://github.com/datastax/python-driver
+      tag: ${{package.version}}
+
+  - name: Python Build
+    uses: python/build-wheel
+
+  - uses: strip
+
+update:
+  enabled: true
+  manual: false
```

Comment:
This should just be 'Apache-2.0', which is the spdx short identifier:
https://spdx.org/licenses/Apache-2.0.html